templates/course-theme/lesson-quiz-notice.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
Cool!

I see the indentation a little bit exaggerated but I think this is the first template I see, so probably that's how it is supposed to look hehe

Reply:
I did a refactor here: https://github.com/Automattic/sensei/pull/4477/commits/b6c7eaf74ba01bf4e2b6f30b98275bfba9909812

In the other templates, we don't use functions, but I don't think it'd be a problem.